index.html

```diff
@@ -41,10 +41,14 @@
                 <img class="hero-img" src="./assets/img/heros/hero-home.jpg" alt="photo of man looking at stars">
             </div>
             <div class="hero-content">
-                <span class="hero-text">
+                <div class="hero-text">
                     <h1 class="hero-h1">Travel</h1>
                     <h2 class="hero-h2"> Who Travels?</h2>
                     <p class="hero-body">It could be you! We're the one stop shop for all time and space travel needs.</p>
+                </div>
+              </div>
+        </section>
+
                 </span>
```

Comment:
This span looks like it should be removed

Reply:
removed